test/e_chronicler_web/controllers/journal_entry_controller_test.exs

```diff
@@ -55,4 +55,27 @@ defmodule EChroniclerWeb.JournalEntryControllerTest do
     assert html_response(conn, 400) =~ JournalEntryController.empty_entry_field_error
   end
 
+
+
+  test "renders form for editing chosen journal_entry", %{conn: conn} do
+    {:ok, journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(@valid_journal_entry)
+    conn = get(conn, Routes.journal_entry_path(conn, :edit, journal_entry))
+    assert html_response(conn, 200) =~ "Edit Journal Entry"
+  end
```

Comment:
I don't think you're testing the flash behavior that you have going for a successfully updated entry.

Reply:
I removed the flash in the name of MVP-ness and getting this done before IPM!